src/applications/form-renderer/actions/form-load/index.js

```diff
@@ -61,6 +62,7 @@ export const fetchFormConfig = (
       const form = findFormByFormId(forms, formId);
       const formConfig = createFormConfig(form);
 
+      dispatch(ombInfoLoaded(form.ombInfo));
       dispatch(formLoadingSucceeded(formConfig));
```

Comment:
Again, see more general discussion. It feels a bit odd to have the OMB data treated separately from the rest of the form data.